### ISSUE TITLE
Proxy for 127.0.0.1

### ISSUE THREAD

**Issue by FJGL** [MarkusBernhardt/proxy-vole#32]
Hi.
Wouldn't be better to return allways DIRECT for xxx://127.0.0.1  without  even checking proxy settings.

Regards

**Comment by flyhard** [comment]
No, I don't think so. Sometimes a developer wants to inspect the requests sent to another app running on localhost and puts a proxy in between the two apps. Always returning DIRECT would be a bad thing in that case.

**Comment by ksmehra** [comment]
i agree with @flyhard apart from the scenario described by him, i have seen ZScalar proxy configuration where there is a PAC file loaded from remote address and the default return clause is 127.0.0.1:<someport> and that actually is the ZScalar proxy agent running on the machine.
the system should report back exactly what is intended by the proxy admin team, these substitutions and transformations do not apply well to real life situations

**Comment by gschnepp** [comment]
This project is dead. Please look at more actual forks like e..g [https://github.com/akuhtz/proxy-vole](url)
